src/Validator/Rules/QueryComplexity.php

```diff
@@ -54,7 +56,8 @@ public function getVisitor(ValidationContext $context)
     {
         $this->context = $context;
 
-        $this->variableDefs     = new ArrayObject();
+        // @phpstan-ignore-next-line Initializing with an empty array does not set the generic type
```

Comment:
I know there were some fixes related to this in phpstan recently. Can you recheck it still applies?

Reply:
Still fails.